http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -467,6 +467,7 @@ class ProxyService(
     "Sec-WebSocket-Version",
     "Sec-WebSocket-Key",
     "Sec-WebSocket-Extensions",
+    "Sec-WebSocket-Protocol",
```

Comment:
Need to filter these `Sec-WebSocket-*` headers otherwise they end up getting set twice and the browser complains.